### DISCUSSION TITLE
Can't find "Include Git LFS objects in archives" under the Archives section.

### DISCUSSION THREAD

**Discussion by HarshGupta-mischief** [community/community#114163] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Can't find "Include Git LFS objects in archives" under the Archives section.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @HarshGupta-mischief! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by EvandroFCosta** [comment]
So is there an anwser?
The github Docs page still mentions the Archive section under Settings, but it doesn't exist? I'm an Owner in an organization and would like to test the option "Include Git LFS objects".
If it is no longer there, where it is?
Will someone update the docs to reflect the change?

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-git-lfs-objects-in-archives-of-your-repository
